plugins/kubernetes/app/models/kubernetes/api/pod.rb

```diff
@@ -48,6 +50,27 @@ def containers
         @pod.spec.containers
       end
 
+      def logs(container)
+        client.get_pod_log(name, namespace, previous: restarted?, container: container)
```

Comment:
This would crash if you tried calling this without first calling `client=`, right?  If you have methods that expect there to be a valid client object, you should probably add it as a parameter to the constructor.